client/cli/src/commands/export_blocks_cmd.rs

```diff
@@ -51,7 +51,10 @@ pub struct ExportBlocksCmd {
 	pub to: Option<BlockNumber>,
 
 	/// Use JSON output rather than binary.
-	#[structopt(long = "json")]
+	#[structopt(long = "json",
+		parse(try_from_str),
+		default_value("true"),
+		default_value_if("output", None, "false"))]
 	pub json: bool,
```

Comment:
That is not intuitive. You should rename this to `binary` and print in binary format if this flag is given.

Reply:
Yes, you're right, defaulting to JSON and requiring a `binary` flag is probably the best thing to do!